### PR TITLE
Add option to use existing origin access identity for s3 origins

### PR DESCRIPTION
**Pull request by jaredbrook**

### CODE REVIEW

File: cloudfront.cfhighlander.rb [hunk 1]
```diff
@@ -8,6 +8,7 @@
 
     origins.each do |id,config|
       ComponentParam "#{id}OriginDomainName"
+      ComponentParam "#{id}OriginAccessIdentityInput"
```
Comment (aaronwalker) [inline cloudfront.cfhighlander.rb:11]: you should add a default value of ''

Comment (aaronwalker) [reply cloudfront.cfhighlander.rb:11]: maybe only add the OAI param if the origin is an S3 bucket

**Review by aaronwalker — CHANGES_REQUESTED** [review]
otherwise looks good

File: cloudfront.cfhighlander.rb (hunk 1 above)
Comment (jaredbrook) [reply cloudfront.cfhighlander.rb]: Thanks @aaronwalker I've made these 2 changes

**Review by Guslington — APPROVED** [review]

**Pull request merged**